.github/workflows/test.yml

```diff
@@ -8,6 +8,14 @@ on:
       - '**'
       - '!doc/**'
 jobs:
+  sanity:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run sanity checks
+        run: |
+          make vendor && make fmt && git diff --exit-code
```

Comment:
It doesn't need to be in this patch, but a `go vet` could be nice too.

Reply:
Sounds reasonable to me - added a TODO comment.